### PR TITLE
Add Support for 7.1 builds

### PR DESCRIPTION
**Pull request by funjoker**

### CODE REVIEW

File: WowPacketParserModule.V7_1_0_22900/WowPacketParserModule.V7_1_0_22900.csproj [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
+       Other similar extension points exist, see Microsoft.Common.targets.
+  <Target Name="BeforeBuild">
+  </Target>
+  <Target Name="AfterBuild">
+  </Target>
+  -->
+</Project>
```
Comment (Keader) [inline WowPacketParserModule.V7_1_0_22900/WowPacketParserModule.V7_1_0_22900.csproj]: new line

Comment (funjoker) [inline WowPacketParserModule.V7_1_0_22900/WowPacketParserModule.V7_1_0_22900.csproj]: autogenerated so donno why no newline

File: WowPacketParserModule.V7_1_0_22900/packages.config [hunk 2]
```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Sigil" version="4.7.0" targetFramework="net452" />
+</packages>
```
Comment (funjoker) [inline WowPacketParserModule.V7_1_0_22900/packages.config]: Missing in other module too :D

**Comment by Nelidon** [comment]
How do you generate opcodes for each version?

**Comment by funjoker** [comment]
Opcodes mostly only change on big patches.
So until 7.2 they won't change. You need to add the future 7.1 versions to the 7.1 module. 
No Parsers added yet, but a fallback with 7.0.3.
Opcodes taken from TC repo

**Pull request closed without merging**